products/sle/main.pm

```diff
@@ -502,7 +507,7 @@ sub load_inst_tests() {
     }
     loadtest "installation/addon_products_sle.pm";
     if (noupdatestep_is_applicable) {
-        if (check_var('ARCH', 'x86_64') && sle_version_at_least('12-SP2') && is_server && (!is_sles4sap || is_sles4sap_standard)) {
+        if (check_var('ARCH', 'x86_64') && sle_version_at_least('12-SP2') && is_server && (!is_sles4sap || is_sles4sap_standard) && !check_var('NOT_INSTALL_TO', 1)) {
```

Comment:
Our test are put under sles12sp2 test, but we have different test scenarios, many of them are not installing machine to sles12sp2, but to others. So if "NOT_INSTALL_TO" is set, it means, we do not install to sles12sp2 in that test. If not set, it means installing to sles12sp2, and should go through those steps under this if.